packages/kamino-sdk/src/services/OrcaService.ts

```diff
@@ -0,0 +1,96 @@
+import { Connection } from '@solana/web3.js';
+import Decimal from 'decimal.js';
+import { estimateAprsForPriceRange, OrcaNetwork, OrcaWhirlpoolClient } from '@orca-so/whirlpool-sdk';
+import axios from 'axios';
+import { OrcaWhirlpoolsResponse } from './OrcaWhirlpoolsResponse';
+import { HubbleConfig, SolanaCluster } from '@hubbleprotocol/hubble-config';
+import { WhirlpoolStrategy } from '../kamino-client';
+import { Scope, ScopeToken } from '@hubbleprotocol/scope-sdk';
+import { Position } from '../whirpools-client';
+import { getKaminoTokenName, getScopeTokenFromKaminoMints } from '../constants';
+import { WhirlpoolAprApy } from './WhirlpoolAprApy';
+import { aprToApy } from '../utils';
+
+export class OrcaService {
+  private readonly _connection: Connection;
+  private readonly _cluster: SolanaCluster;
+  private readonly _config: HubbleConfig;
```

Comment:
extremely uncomfortable with spreading these god objects everywhere with every new PR. you just need a getPrice(mint: Pubkey) -> number api so you can get rid of all of these dependencies here and also in the defiLlama issue. because we're not doing this we're creating these impossible to break out dependencies, esp if we need to break out kamino sdk from hubble sdk

Reply:
you already have a Mint->String mapping, you can just call scope as an api and get rid of this dependency and get rid of `ScopeToken` as a type dependency.

Reply:
The original goal of HubbleConfig was to have one npm package that serves as the only source of truth for all of our configs (Hubble, Scope, Kamino,..). This makes it much easier to maintain as we only have to update the one npm package. Then other npm packages get updated automatically as well: hubble-sdk, kamino-sdk, scope-sdk,.. 
This way we bump versions in hubble-public-api/hubble-prom-metrics/hubble-historian and other tools we have, instead of having to maintain x amount of configs.

I do agree, however, that it's becoming a bit too big and it can make testing a bit more difficult, but that can be solved by making the config overridable in the constructor of our SDKs so we can use dependency injection in our integration/unit tests and easily inject our own fixtures.

I think the usage of this package brings way too many positives to the SDK and at the end of the day you'll need to have a mint to token map anyway, doesn't it make sense to only have one in the hubble-config package and only update that one, instead of having to do it all over the place?